src/impl/tpls/KokkosBlas_tpl_spec.hpp

```diff
@@ -18,11 +61,18 @@ struct CudaBlasSingleton {
 
 }
 }
-#endif
+#endif // KOKKOSKERNELS_ENABLE_TPL_CUBLAS
+
+#ifdef KOKKOSKERNELS_ENABLE_TPL_LAPACK
+#include "lapacke.h" // LAPACKE_
+#endif // KOKKOSKERNELS_ENABLE_TPL_LAPACK
 
+// If LAPACK TPL is enabled, it is preferred over magma's LAPACK
 #ifdef KOKKOSKERNELS_ENABLE_TPL_MAGMA
 #include "magma_v2.h"
+#ifndef KOKKOSKERNELS_ENABLE_TPL_LAPACK
 #include "magma_lapack.h"
```

Comment:
I think ```magma_v2.h``` would be enough. Please double check.

Reply:
TODO: see if `gesv` requires `magma_lapack.h`.